SU-HSDO/suhumsci
# ARCH-58 Basic Page Content type and a couple paragraphs

# READY FOR REVIEW
- I dont know a great way to remove the display settings from the module. either it lives here or it lives in the profile. both have the same implications with over-ridden state. but from the sounds of H&S they mainly want a "time in point" install. and won't want updates.

# Summary
- Basic Page content type
- Some initial paragraph types.
- Menu block fixes
- new regions to easily place content (is this good?)

# Needed By (Date)
- Whenever

# Urgency
- low

# Steps to Test

1. Checkout this branch,
2. `composer install --prefer-source` (If it has issues because of a drupal/core patch, remove the drupal/core patch from `blt/composer.required.json`)
3. `drush si -y`
4. log in and ensure the basic page content type exists
5. ensure hero image and text area paragraphs exist
6. optionally enable the other paragraph types: hs_slideshow, hs_view
7. Create a basic page and fill out appropriate fields
8. link it into the menu
9. create a second basic page and put it in the menu under the first
10. ensure the menu block appears in the left sidebar area.
11. additional responsive and centering styling needed in the future.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
